rad/src/operators/array.rs

```diff
@@ -84,18 +83,37 @@ pub fn get_bytes(input: &RadonArray, args: &[Value]) -> Result<RadonBytes, RadEr
     let item = get(input, args)?;
     item.try_into()
 }
-pub fn get_integer(input: &RadonArray, args: &[Value]) -> Result<RadonInteger, RadError> {
-    let item = get(input, args)?;
-    item.try_into()
-}
+
+/// Try to get a `RadonFloat` from a position in the input `RadonArray`, as specified by the first
+/// argument, which is used as the positional index.
 pub fn get_float(input: &RadonArray, args: &[Value]) -> Result<RadonFloat, RadError> {
-    let item = get(input, args)?;
-    item.try_into()
+    get_numeric_string(input, args)?.try_into()
 }
+
+/// Try to get a `RadonInteger` from a position in the input `RadonArray`, as specified by the first
+/// argument, which is used as the positional index.
+pub fn get_integer(input: &RadonArray, args: &[Value]) -> Result<RadonInteger, RadError> {
+    get_numeric_string(input, args)?.try_into()
+}
+
 pub fn get_map(input: &RadonArray, args: &[Value]) -> Result<RadonMap, RadError> {
     let item = get(input, args)?;
     item.try_into()
 }
+
+/// Try to get a `RadonTypes` from a position in the input `RadonArray`, as specified by the first
+/// argument, which is used as the positional index.
+///
+/// This simply assumes that the element in that position is a number (i.e., `RadonFloat` or
+/// `RadonInteger`). If it is not, it will fail with a `RadError` because of `replace_separators`.
+fn get_numeric_string(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
+    let item = get(input, &args[..1])?;
+
+    let (thousands_separator, decimal_separator) = string::read_separators_from_args(args, 1);
```

Comment:
The skip argument seems redundant because you can pass a sub-slice, so this should be equivalent:

```suggestion
    let (thousands_separator, decimal_separator) = string::read_separators_from_args(&args[1..], 0);
```